spec/lib/account_deleter_spec.rb

```diff
@@ -107,11 +108,18 @@
     end
   end
 
+  describe "#delete_user_invitation_code" do
+    it 'deletes user invitation code' do
+      expect(bob.invitation_code).to receive(:destroy)
+      @account_deletion.delete_user_invitation_code
+    end
+  end
+
   describe "#delete_standard_person_associations" do
     before do
       @account_deletion.person = bob.person
     end
-    it 'removes all standard person associaltions' do
+    it 'removes all standard person associations' do
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.